peachjam_search/migrations/0001_es_mapping_add_provisions.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.2.25 on 2024-04-26 17:17
+
+from django.db import migrations
+
+
+def forwards(apps, schema_editor):
+    from django.conf import settings
+    from django_elasticsearch_dsl.registries import registry
+
+    if not settings.DEBUG:
+        for ix in registry.get_indices():
```

Comment:
this allows us to add the new field to the mapping (which is allowed without having to destroy and re-create the mapping).

we only do it when DEBUG = False because we don't have an ES to talk to when testing.

Reply:
neat